routes/index.js

```diff
@@ -6,13 +6,16 @@ router.get('/', function(req, res, next) {
   res.render('index', {
     title: 'chubbo-chat',
     //pass src for page bkground only needed on landing
-    background: '/javascripts/floatingDots.js'
+    script: '/javascripts/floatingDots.js'
```

Comment:
I did it like this because on the dashboard page it gets an error, since the element its looking for only exists on the landing page: http://puu.sh/pUPyI/b17cf9a152.png 
But I guess it doesn't really matter if that error comes up, it doesn't affect anything